conference_system/README.md

```diff
@@ -0,0 +1,127 @@
+<h1 id="conference" align="center">ConferenceSystem</h1>
+
+<p align="center">
```

Comment:
Can we use Markdown instead of HTML for our documentation?

Reply:
Using markdown is good to me. Doc is still under developing, so I believe we can hang this pr until this weekend. Till then, we will get a much better doc to merge.